src/pages/Governance/Proposal/Header.tsx

```diff
@@ -23,14 +23,40 @@ function TitleComponent({proposal}: {proposal: Proposal}): JSX.Element {
 
 function StatusComponent({proposal}: {proposal: Proposal}): JSX.Element {
   return (
-    <ProposalStatusTooltip>
-      <Box sx={{display: "flex", alignItems: "center", gap: 1}}>
-        <StatusIcon status={proposal.status} />
-        <Typography variant="subtitle1" color={getStatusColor(proposal.status)}>
-          {proposal.status}
-        </Typography>
-      </Box>
-    </ProposalStatusTooltip>
+    <Stack direction="row" spacing={2}>
+      <ProposalStatusTooltip>
+        <Box sx={{display: "flex", alignItems: "center", gap: 1}}>
+          <StatusIcon status={proposal.status} />
+          <Typography
+            variant="subtitle1"
+            color={getStatusColor(proposal.status)}
+          >
+            {proposal.status}
+          </Typography>
+        </Box>
+      </ProposalStatusTooltip>
+      {ExecutionTimeComponent(proposal)}
+    </Stack>
+  );
+}
+
+function ExecutionTimeComponent(proposal: Proposal) {
+  // TODO: remove proposal.resolution_time_secs null check after AIT3,
+  // as we shouldn't have null resolution_time_secs by then
+  if (!(proposal.is_resolved && proposal.resolution_time_secs)) {
```

Comment:
Why not if (!proposal.is_resolved || !proposal.resolution_time_secs) ? So if the proposal has not been resolved or the resolution time secs is not available. This makes more sense to me

Reply:
I think they're essentially the same. But I'll change it for better readability.